transform/mattermost-analytics/models/staging/mm_calls_test_go/_mm_calls_test_go__models.yml

```diff
@@ -0,0 +1,28 @@
+version: 2
+
+models:
+  - name: stg_mm_calls_test_go__tracks
+    description: |
+      Reconstructed `tracks` table using custom properties expected to be in the events.
+
+    columns:
+      - name: event_id
```

Comment:
There are columns in the model that are not listed here. They need to be added.

Reply:
Yes, this was weird, maybe calls work differently? we can check with Claudio later

Reply:
Added the missing columns